app/utils/PriceUtils.java

```diff
@@ -1,55 +1,52 @@
 package utils;
 
-import com.google.common.base.Optional;
-import io.sphere.client.model.Money;
-import io.sphere.client.shop.model.Price;
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.products.Price;
+import io.sphere.sdk.utils.MoneyImpl;
 
-import java.text.NumberFormat;
+import javax.money.MonetaryAmount;
+import javax.money.format.MonetaryFormats;
 import java.util.Locale;
-
-import static java.math.BigDecimal.ROUND_HALF_EVEN;
-import static java.util.Currency.getInstance;
+import java.util.Optional;
 
 public final class PriceUtils {
 
     private PriceUtils() {
     }
 
-    public static String format(final Money money) {
-        final String amount = NumberFormat.getInstance(Locale.GERMANY).format(money.getAmount());
-        final String currency = getInstance(money.getCurrencyCode()).getSymbol(Locale.GERMANY);
-        return amount + " " + currency;
+    public static String format(final Price price) {
+        return format(price.getValue());
     }
 
-    public static double monetaryAmount(final Money money) {
-        return money.getAmount().setScale(2, ROUND_HALF_EVEN).doubleValue();
+    public static String format(final Cart cart) {
```

Comment:
http://sphereio.github.io/sphere-jvm-sdk/javadoc/v1.0.0-M17/io/sphere/sdk/meta/FormattingDocumentation.html#money

Do not use `doubleValue()` this can cause issues.